addons/hr_timesheet/views/hr_timesheet_portal_templates.xml

```diff
@@ -44,8 +44,7 @@
                             <th t-if="not groupby == 'project'">Project</th>
                             <th t-if="not groupby == 'task'">Task</th>
                             <th>Description</th>
-                            <th t-if="is_uom_day" class="text-end">Days Spent</th>
-                            <th t-else="" class="text-end">Hours Spent</th>
+                            <th class="text-end" id="t_label">Time Spent</th>
```

Comment:
Is it really needed? 🧐 

Ideally, we should avoid using `id` in backend views because the id should be unique in the DOM.